src/base/projects/Readme.svelte

```diff
@@ -14,9 +14,8 @@
 <style>
   article {
     padding: 2rem;
-    max-width: 64rem;
```

Comment:
I thought this was a deliberate change by @cloudhead: https://github.com/radicle-dev/radicle-interface/commit/771ed2aea7b74e7f1108224a40d43e5fed967aa9.

What's the reason for the change?

I think it was so that the RIPs look better:
http://127.0.0.1:3000/seeds/clients.radicle.xyz/rad:git:hnrkx7ruakgb9ozzi7atwb1rh6moq8pjonp6o/tree/8cd1561688462e2fcb234956b6f5916c42bdf4fa/0001-heartwood.md

Your suggested change:
<img width="1840" alt="Screenshot 2022-11-07 at 13 31 35" src="https://user-images.githubusercontent.com/158411/200311151-b245c87b-0a3b-48bc-bf59-e4c707e490e7.png">

vs production

<img width="1840" alt="Screenshot 2022-11-07 at 13 31 44" src="https://user-images.githubusercontent.com/158411/200311217-4f7bca0f-70b5-4f52-8017-d4e1ccff99e9.png">

Reply:
Good catch didn't know that, the reason was that due to using the `file-header` now from the `Blob` component that uses `width: 100%` we had a header that went until the right side and the markdown stopped at `64rem` will provide a better solution

Reply:
217c87628e6eb9de1351c31e2477ad84e64a79a2